WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanStatusViewModel.swift

```diff
@@ -203,13 +203,14 @@ struct JetpackScanStatusViewModel {
 
     // MARK: - Localized Strings
     private struct Strings {
-        static let noThreatsTitle = NSLocalizedString("Don’t worry about a thing", comment: "Title for label when there are no threats on the users site")
-        static let noThreatsDescriptionFormat = NSLocalizedString("The last Jetpack scan ran %1$@ and everything looked great.\n\nRun a manual scan now or wait for Jetpack to scan your site later today.", comment: "Description for label when there are no threats on a users site and how long ago the scan ran")
-        static let noThreatsDescription = NSLocalizedString("The last Jetpack scan completed and everything looked great.\n\nRun a manual scan now or wait for Jetpack to scan your site later today.", comment: "Description that informs for label when there are no threats on a users site")
+        static let noThreatsTitle = NSLocalizedString("Risks Removed", comment: "Title for label when there are no threats on the users site")
```

Comment:
This looks odd to me. What if there was never a threat/risk there? Can we say we removed something?

Reply:
@osullivanchris JFYI: I was talking to @emilylaguna and we decide to revert this one. "Risks Removed" sounds very vague and it might be misleading: perhaps no risks were there at all.

Can you double-check with the editorial team? Once we have the new string we can update it, for now, we'll move on keeping the previous one.

Reply:
@leandroalonso  Hey its hard for me to follow the code here sorry! Can you tell me what string we're talking about and what the before and after was? Thanks!

Reply:
@osullivanchris Sorry he's talking about the change from `Don’t worry about a thing` to `Risks Removed`

Reply:
@emilylaguna thanks for clarifying! Yes I think I agree. I would only have this copy right after risks are removed. It sounds like we cannot be certain of that, so the more generic 'Don't worry about a thing' is ok for now. I still think we can improve that string to be less fluffy, but at least its not misleading.